templates/autolabel_tma_cores.ipynb

```diff
@@ -195,6 +195,11 @@
     "* The selected proposed FOV and its mapped automatically-generated FOV are colored a darker shade.\n",
     "* The `Manually-defined FOV` menu can be used to visualize current mappings to automatically-generated FOVs.\n",
     "* The `Automatically-generated FOV` menu is used for re-mapping a FOV created by this script to one of yours.\n",
+    "* Additional validation parameters:\n",
```

Comment:
See if you can take another pass at these descriptions. Right now these feel like you're trying to write a manual. Reword them as if you're explaining to someone sitting next to you what each of these things does.